tensorflow_datasets/image/downsampled_imagenet.py

```diff
@@ -0,0 +1,136 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Downsampled Imagenet dataset."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+
+from tensorflow_datasets.core import api_utils
+import tensorflow_datasets.public_api as tfds
+
+# From https://arxiv.org/abs/1601.06759
```

Comment:
I think it's unnecessary.

Reply:
Actually I followed [this](https://github.com/tensorflow/datasets/blob/90c0b38438afcb56044f13b11a426d013d0c8848/tensorflow_datasets/image/cycle_gan.py#L29).

Reply:
Already written [there](https://github.com/tensorflow/datasets/blob/90c0b38438afcb56044f13b11a426d013d0c8848/tensorflow_datasets/image/cycle_gan.py#L41) so it be written twice. It's also unnecessary.

Reply:
Okk, I see.